libr/io/p/io_null.c

```diff
@@ -0,0 +1,129 @@
+/* radare - LGPL - Copyright 2017 - condret, pancake */
```

Comment:
what is exactly null:// useful for? can you put this in a separate PR for discussion